app/scripts/modules/core/application/config/serializeApplicationSection.directive.html

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col-md-7">
+    <h4>Serialize Application</h4>
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <p>
+          Serializing the application will save the current state of the infrastructure using Terraform in a bucket. <br>
+          Currently only GCE and GCS are supported.
+        </p>
+
+        <button class="btn btn-link" ng-click="vm.serializeApplication()"><span
+          class="glyphicon glyphicon-cloud-upload"></span> Serialize Application
```

Comment:
Ah I see what you mean... it's kind of funny that we're moving the definition from one cloud to another...